building-and-packaging/material/examples/shared-and-static-c++/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.10)
+
+# set the project name
+project(Shared-Libraries-C++)
+
+# add the executable
+add_executable(shared-libraries-example main.cpp)
```

Comment:
I guess you could cut this one. Will be covered in the cmake part. Or what exactly did you want to showcase here?

Reply:
I do not plan to use this anymore. I started with the CMake file, but found the Makefile more practical and simpler.